### PR TITLE
Update models.yml for multiple votes per option

### PR DESCRIPTION
**Pull request by MJJojo97**
See https://github.com/OpenSlides/openslides-backend/pull/1251

### CODE REVIEW

**Review by ostcar — APPROVED** [review]

File: global/meta/models.yml [hunk 1]
```diff
@@ -2654,11 +2654,17 @@ poll:
...
     restriction_mode: A
   max_votes_amount:
     type: number
     default: 1
+    minimum: 1
     restriction_mode: A
+  max_votes_per_option:
+    type: number
+    default: 1
+    minimum: 1
```
Comment (ostcar) [inline global/meta/models.yml:2667]: `restriction_mode: A` is missing

**Review by ostcar — CHANGES_REQUESTED** [review]

File: global/meta/models.yml (hunk 1 above)
Comment (MJJojo97) [reply global/meta/models.yml:2667]: Oh yes. I am very sorry about that!

**Comment by ostcar** [comment]
Ohh, I see there is a check, that the example_data is valid. I did not realise that. This is strange, because your field is not required. So why do you have to add it to the example data?

Anyway, could you add it?

**Comment by MJJojo97** [comment]
Sorry for the inconvenience! I didn't have the full overview on what make-tasks I have to run. Otherwise, I would have seen that problem before.

I hope that everything is working now.

**Comment by jsangmeister** [comment]
> Ohh, I see there is a check, that the example_data is valid. I did not realise that. This is strange, because your field is not required. So why do you have to add it to the example data?

Because it has a a default value - and yes, I know how strange that sounds :D

> Sorry for the inconvenience! I didn't have the full overview on what make-tasks I have to run. Otherwise, I would have seen that problem before.

No problem, it's a lot one has to take care of when changing the models definition. You can be very happy that no migration is required here :D

**Review by jsangmeister — APPROVED** [review]

**Pull request merged**